Website updates - Tag truncation (#2694)
hashicorp/design-system: website/docs/components/tag/partials/guidelines/guidelines.md

## Content

When content within the tag is not user-generated, it should be concise and consistent with the category name.

### Truncation

> Tags have a max character limit of about 20 and if the text is longer it will truncate. The full text is displayed in a [Tooltip](/components/tooltip) when the user hover or focuses the Tag's text. The Tooltip is above the Tag by default, but this can be customized.

Reviewer: ```suggestion
Tags can display about 20 characters maximum before the text becomes automatically truncated. When truncated, the full text will display in a [Tooltip](/components/tooltip) that appears when the user hovers or focuses the Tag. This Tooltip displays above the Tag by default; however, the placement can be customized.
```

Reviewer 2: I would give the specific length then the approximate after that so readers can see the measurement first then ground that in their experience of the character count.

```suggestion
Tags have a max length of 160px (approx. 20 characters) after which it will be automatically truncated. When truncated, the full text will display in a [Tooltip](/components/tooltip) that appears on hover or focus. This Tooltip displays above the Tag content by default; however, the placement can be customized.
```

Author: @LilithJames-HDS we removed the mention of the max width because it's not technically accurate (160px for non-dismiss variants and 166px for dismiss variants), which is why we only talked about character count.

> The text container of the tag has a max-width of 150px. The text will truncate if it goes beyond that limit. A tooltip will display the full text when the user hovers or focuses on the truncated text. The tooltip will be placed at the top by default, but this can be customized.

Reviewer 3: ```suggestion
Tags have a max-width of 150px and if the text is longer it will truncate. The full text is displayed in a [Tooltip](/components/tooltip) when the user hover or focuses the Tag. The Tooltip is above the Tag by default, but this can be customized.
```

Just a suggestion, trying to shift the language to be more active.

> The text container of the tag has a max-width of 150px and will truncate text if any content overflows beyond that limit. If truncation is present, a tooltip will appear with the full text, on hover or focus of the text container. By default, the tooltip's placement will be top, however, this can be customized if necessary.

Reviewer 4: [suggetion] smaller sentences and more active voice

```suggestion
The text container of the tag has a max-width of 150px. The text will truncate if it goes beyond that limit. A tooltip will display the full text when the user hovers or focuses on the truncated text. The tooltip will be placed at the top by default, but this can be customized.
```

![](/assets/components/tag/tag-truncation-tooltip.png)

!!! Info

> It is only possible to change the Tooltip placement in Ember. To keep the Figma component simple, there is only one placement (top).

Reviewer: ```suggestion
Tooltip placement can only be customized in the Ember component. For simplicity, the Figma component only includes top placement.
```

> Changing the tooltip placement is only possible in the Ember component. In order to keep the Figma component simple, we are only showing one type of placement.

Reviewer 3: ```suggestion
It is only possible to change the Tooltip placement in Ember. To keep the Figma component simple, there is only one placement (top).
```

The text container of the tag has a max-width of 150px and will truncate text if any content overflows beyond that limit. If truncation is present, a tooltip will appear with the full text, on hover or focus of the text container. By default, the tooltip's placement will be top, however, this can be customized if necessary.

![](/assets/components/tag/tag-truncation-tooltip.png)

!!! Info

> Updating the tooltip placement is only possible in the ember component. In order to keep the Figma component simple, we are only showing one type of placement for the tooltip.

Reviewer: ```suggestion
Changing the tooltip placement is only possible in the Ember component. In order to keep the Figma component simple, we are only showing one type of placement.
```
